backend/accessibility_templates/aa-004.xml

```diff
@@ -0,0 +1,21 @@
+<AccessibilityAttributes>
+ <SubtitleAttributes>
+  <Carriage href="urn:tva:metadata:cs:SubtitleCarriageCS:2023::3"/>
+  <Coding href="urn:tva:metadata:cs:SubtitleCodingFormatCS:2023:4"/>
```

Comment:
The coding format should be DVB DASH and HbbTV compliant, in this case EBU-TT-D in the version EBU-TT-D 1.0.1 with the termin ID 3.2.2 As EBU-TT-D is also conforms to IMSC a second CodingFormat element should be added with the termID 3.6.2.2 for IMSC 1.0.1.